readthedocs/restapi/templates/restapi/log.txt

```diff
@@ -0,0 +1,13 @@
+RTD build information start
+Build: {{ build.pk }}
+Project: {{ build.project.slug }}
+Version: {{ build.version.verbose_name }} ({{ build.commit }})
+Date: {{ build.date }}
+Success: {{ build.success }}
+RTD build information end
+
+{% for command in build.commands.all %}
```

Comment:
Are these command already sorted? What's the default sorting?

We should probably force the sorting here to avoid future problems using `order_by` or similar.

Reply:
Yes and no :)

That sorting is the default sorting and it's not the one that we want (since it will show the latest first because of the `-`), but on the other hand, this one applies to the `Build` model, not the `BuildCommandResult`

Anyway, I just checked this and it's what we need: sorted by `start_time`: https://github.com/rtfd/readthedocs.org/blob/7812234dbaecbc37e899e2ae73fce7c3de277433/readthedocs/builds/models.py#L511

So, nothing to do here I'd say

Reply:
Yeah, that was the meta class that I saw the other time, I put the wrong model here, sorry for the confusion.